USAGE.md

```diff
@@ -0,0 +1,15 @@
+# Usage
+
+## For basic interval arithmetic
+
+```
+>>> from pymath.interval import interval
+
```

Comment:
Python repl has 3 different types of lines. ">>> " for the start of a line of code. "... " for every line after (e.g. inside of a for-loop). And no line prefix for stdout/stderr. So comments should be written as ">>> # Comment" instead of "# Comment".